docs/packaging-applications/build-servers/tfs-azure-devops/using-octopus-extension/extension-compatibility.md

```diff
@@ -28,45 +28,15 @@ TFS 2017 Update 1 is technically supported with version 2.0.199 of the extension
 
 Any version older than TFS 2015 Update 2 is not supported by any extension version.
 
-You may have success manually installing the task by following [these instructions](manually-install-the-build-task.md).
-
 ### Build information compatibility {#build-information-compatibility}
 
-When passing [build information](docs/packaging-applications/build-servers/build-information/index.md) to Octopus from TFS, you may encounter issues when trying to use the build link generated by the Octopus TFS extension.
+When passing [build information](docs/packaging-applications/build-servers/build-information/index.md) to Octopus from TFS, you may encounter issues when trying to use the build link generated by the TFS extension.
 
-Specifically, the build link may return a `404 (Not Found)` error when viewed. The cause for the issue is believed to be the result of a change to the format of the build URL supported by TFS/Azure DevOps.
+Specifically, the build link may return a `404 (Not Found)` error when viewed. The cause for the issue is believed to be the result of a change to the format of the build URL supported by Azure DevOps.
 
 The Build information step in the Octopus TFS plugin expects the build to be viewed using a URL like this: 
```

Comment:
Consistency
> ~Octopus TFS~ Azure DevOps plugin

This also says `plugin`, whereas above it used `extension`, we should be consistent.